### ISSUE TITLE
Infinite Loading: Can't Find Lance

### ISSUE THREAD

**Issue by CWolfs** [CWolfs/MissionControl#425]
Infinite Loading: Can't Find Lance bug.

Full description provided by @Haree78 in PR #423.

**Comment by CWolfs** [comment]
In `develop` with PR #423.

**Comment by CWolfs** [comment]
This caused an issue in RT due to their unlimited lance difficulty.

Expand this out to a `settings.json` setting with a `Enable` and `MaxDiffDivergence` setting.

**Comment by CWolfs** [comment]
In `develop`.

**Comment by CWolfs** [comment]
`v1.3.0` will have the following settings:

```json
  "Misc": {
    "LanceSelectionDivergenceOverride": {
      "Enable": true,
      "Divergence": 20
    }
  }
```

**Comment by CWolfs** [comment]
The exposed settings were released out as part of `v1.2.1`.
